examples/hello-world-pyprojecttoml/rust/lib.rs

```diff
@@ -0,0 +1,35 @@
+use pyo3::prelude::*;
+use std::env;
+
+/// Formats the sum of two numbers as string.
+#[pyfunction]
+fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
+    Ok((a + b).to_string())
+}
+
+/// Calls Python (see https://pyo3.rs for details)
+#[pyfunction]
+fn demo(py: Python) -> PyResult<()> {
```

Comment:
There seems to be a warning here because `py` is unused. Sorry I am not experienced in `Rust`, so I got confused by the following `Python::with_gil(|py| ...` statement.

```suggestion
fn demo(_py: Python) -> PyResult<()> {
```

If a maintainer would like to commit this to the PR, that would trigger the CI. Otherwise I can add it myself and wait for someone to trigger the CI.

Reply:
No problem, I've force-pushed again. (Removed `Python::with_gil` bit, as we can use `py` directly from the argument.)

Reply:
Ah! Perfect, thank you @davidhewitt.

Still baby steps on Rust :P